tsconfig.json

```diff
@@ -100,12 +94,15 @@
      */
     "src/constants/**/*",
     "src/server-middleware/healthcheck.js"
+    /** ******** */
```

Comment:
Do you know where this came from?

Reply:
yeah :), comment-json removes all white lines from tsconfig file after stringifing it back to json.
i add this line as a seperator(before there was a white line at line 97) between the section of non components and the components that will be add in the future right after this section, if one uses the command line to create the component.
i can remove it if you``ll think its not necessary.

Reply:
Ah, I see! It might be better to replace the string of `*` with actual text, otherwise it looks like something prime for deletion by someone with less context down the road :slightly_smiling_face: